src/BuildToolchains.jl

```diff
@@ -299,3 +310,168 @@ function cargo_config_file!(dir::AbstractString, platform::AbstractPlatform;
                   """)
     end
 end
+
+function bazel_cpu(p::AbstractPlatform)
+    if arch(p) == "x86_64"
+        return "k8"
+    else
+        return arch(p)
+    end
+end
+
+# TODO distinguish between clang and gcc toolchains?
+# TODO in _impl, get cpu, target_libc, abi_version and abi_libc_version from host + env info
+# TODO add `-lm` to `link_libs` on Reactant
+function toolchain_file(bt::Bazel, p::AbstractPlatform, host_platform::AbstractPlatform; is_host::Bool=false, clang_use_lld::Bool=false)
+    target = triplet(p)
+    aatarget = aatriplet(p)
+    host_target = triplet(host_platform) # TODO fix this
+
+    return """
+    load("@rules_cc//cc:defs.bzl", "cc_toolchain")
+    load(
+        "@bazel_tools//tools/cpp:cc_toolchain_config_lib.bzl",
+        "action_config",
+        "artifact_name_pattern",
+        "feature",
+        "feature_set",
+        "flag_group",
+        "flag_set",
+        "tool",
+        "tool_path",
+        "variable_with_value",
+        "with_feature_set",
+    )
+
+    def _impl(ctx):
+        return cc_common.create_cc_toolchain_config_info(
+            ctx = ctx,
+            features = [...], # TODO
+            cxx_builtin_include_directories = [
+                # TODO maybe requires to add them to `compile_flags` with `-isystem`
+                # TODO are these really required?
+                # "/opt/$(aatarget)/lib/gcc/$(aatarget)/10.2.0/include",
+                # "/opt/$(aatarget)/lib/gcc/$(aatarget)/10.2.0/include-fixed",
+                # TODO these paths should use %sysroot% instead of hardcoding the sysroot
+                # "/opt/$(aatarget)/$(aatarget)/include",
+                # "/opt/$(aatarget)/$(aatarget)/sys-root/usr/include",
+                # "/opt/$(aatarget)/$(aatarget)/include/c++/10.2.0",
+                # "/opt/$(aatarget)/$(aatarget)/include/c++/10.2.0/$(aatarget)",
+                # "/opt/$(aatarget)/$(aatarget)/include/c++/10.2.0/backward",
+                # "/opt/$(aatarget)/$(aatarget)/include/c++/10.2.0/parallel",
+                "%sysroot%/../include",
+                "%sysroot%/usr/include",
+                "%sysroot%/../include/c++/10.2.0",
+                "%sysroot%/../include/c++/10.2.0/$(aatarget)",
+                "%sysroot%/../include/c++/10.2.0/backward",
+                "%sysroot%/../include/c++/10.2.0/parallel",
```

Comment:
Hardcoding paths like this doesn't seem good, also because they're specific to the GCC version used.